emission/analysis/classification/inference/mode.py

```diff
@@ -79,29 +146,39 @@ def getModeQuery(groundTruthMode):
 
   @staticmethod
   def getSectionQueryWithGroundTruth(groundTruthMode):
-    return {"$and": [{'type': 'move'},
+    # return {"$and": [{'type': 'move'},
+    return {"$and": [{"$or": [{'type': { '$exists' : False }}, {'type' : 'move'}]},        
+                     ModeInferencePipeline.getModeQuery(groundTruthMode)]}
+
+  @staticmethod
+  def getSectionQueryWithGroundTruthUser(groundTruthMode, uuid):
+     return {"$and": [{"$or": [{'type': { '$exists' : False }}, {'type' : 'move'}]},        
```

Comment:
Yes, I didn't create this function, I just changed the part about the type being move. I don't know how it relates to a specific user either, so I will make it so.